docs/spec_draft.md

```diff
@@ -1443,6 +1445,169 @@ specification. Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.scatter
+
+### Semantics
+
+Generate `results` which is the values of the `inputs` operand, with several
+slices at indices specified by `scatter_indices`, updated with the values in
+`updates` using `update_computation`.
+
+The following diagram shows how elements in `updates[k]` map on elements in
+`inputs[k]` using a concrete example. The diagram picks a few example
+`updated[k[` indices and explains in detail which `inputs[k]` indices they
+correspond to.
+
+<img align="center" src="spec_images/scatter.png" />
+
+More formally, for any `k`, `updates[k][update_index] =
+inputs[k][input_index]` where:
+  * `update_scatter_dims` = [`d` for `d` in `axes(updates[k])` and `d` not in
+    `update_window_dims`].
+  * `update_scatter_index` = [`update_index[d]` for `d` in
+    `update_scatter_dims`].
+  * `start_index` =
+      * `scatter_indices[bi0, ..., :, ..., biN]` where `bi` are individual
```

Comment:
`si0`, ..., `siN` given that we're referring to "scatter indices" and not "batch indices" (from gather)?